### PR TITLE
Update header.html

### PR DESCRIPTION
**Pull request by jamesxsu**
Fixed the UI bug where the contest wouldn't automatically make the user leave the contest once it has ended.

### CODE REVIEW

**Comment by espidev** [comment]
I think you're forgetting the redirect

**Comment by jamesxsu** [comment]
> I think you're forgetting the redirect

What do you mean redirect? @espidev I'm new to this so how would I do the redirect?

**Comment by espidev** [comment]
> What do you mean redirect? @espidev I'm new to this so how would I do the redirect?

You just created a variable called leaveURL, you need to actually call a redirect in the browser: https://www.w3schools.com/howto/howto_js_redirect_webpage.asp

**Comment by espidev** [comment]
So this is probably enough `window.location.replace(leaveURL);`

File: src/main/resources/templates/fragments/header.html [hunk 1]
```diff
@@ -56,6 +56,11 @@
         function updateCountDown() {
             let origCountdownTime = /*[[${currentContest.participants.get(#authentication.principal.id).getSubmissionEndTime(currentContest)}]]*/;
             let nTime = origCountdownTime - Date.now();
+            
+            if(nTime < 0) {
+                let leaveURL= "/contest/" + /*[[${currentContest.name}]]*/ + "/leave";
```
Comment (espidev) [inline src/main/resources/templates/fragments/header.html:61]:
```suggestion
                let leaveURL= "/contest/" + /*[[${currentContest.name}]]*/ + "/leave";
                window.location.replace(leaveURL);
```
should work

**Comment by jamesxsu** [comment]
Ok I have updated the file, I think its correct, merge?

**Comment by espidev** [comment]
Looks good

**Pull request merged**